lib/client/client.js

```diff
@@ -1469,8 +1469,9 @@ Client.prototype._sendSocket = function _sendSocket(message,
     timer = setTimeout(onRequestTimeout, self.timeout);
   }
 
-  if (log.trace())
-    log.trace('sending request %j', message.json);
+  if (log.trace()) {
+    log.trace('sending request %j', message.safeToString());
```

Comment:
Where is the `safeToString` method defined? I don't see it when I search the existing code base.